walkthroughs/week-2-server/examples/random-quotes/src/main/webapp/index.html

```diff
@@ -10,7 +10,7 @@
     <div id="content">
       <h1>Random Quotes</h1>
       <p>Click here to see a random quote:</p>
-      <button onclick="getRandomQuote()">Fetch a quote!</button>
+      <button onclick="getRandomQuoteUsingAsyncAwait()">Fetch a quote!</button>
```

Comment:
Remove this change. This is one of the examples and you probably accidentally edited it.

Reply:
done